Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "tree-sitter-scala"
 description = "scala grammar for the tree-sitter parsing library"
-version = "0.19.0"
+version = "0.0.1"
```

Comment:
version number is 0.19.0 in package.json:
https://github.com/tree-sitter/tree-sitter-scala/blob/0ceab66a43914320da893199ecd760473c043769/package.json#L3

Is there a reason to roll this back to 0.0.1 for Rust binding specifically?

Reply:
I kinda forgot why I did this, it's been awhile. I think I just saw a bunch of other tree-sitter repos keep the version number at 0.0.1. Which I think is bad anyways. A good system would probably be to just keep this up with the tree-sitter version.

It also could have been something that was auto-generated. I forget if I changed this file manually or not.